Kudu.Core/K8SE/K8SEDeploymentHelper.cs

```diff
@@ -161,17 +161,41 @@ private static string RunBuildCtlCommand(string args, string msg)
 
         public static string GetAppName(HttpContext context)
         {
-            var appName = context.Request.Headers["K8SE_APP_NAME"].ToString();
-
+            var appName = context.Request.Headers["k8se_app_name"].ToString();
+            if (string.IsNullOrEmpty(appName))
+            {
+                appName = "lima";
+            }
             if (string.IsNullOrEmpty(appName))
             {
                 context.Response.StatusCode = 401;
                 // K8SE TODO: move this to resource map
                 throw new InvalidOperationException("Couldn't recognize AppName");
             }
+            Console.WriteLine($"Get App name: {appName}");
             return appName;
         }
 
+        public static string GetExtensionName(HttpContext context)
+        {
+            var extensionName = "zuh3-appservice-extension-k8se-build-service";
+
+            try
+            {
+                extensionName = System.Environment.GetEnvironmentVariable("POD_DEPLOYMENT_NAME").ToString();
+            }
+            catch (Exception)
```

Comment:
Suggestion (practice): We should try not to throw exception as much as we can.

For this part, we can implement like this
```
var extensionName = System.Environment.GetEnvironmentVariable("POD_DEPLOYMENT_NAME");
if (string.IsNullOrEmpty(extensionName))
{
    Console.WriteLine($"Not found env var POD_DEPLOYMENT_NAME. Default to {extensionName}");
    extensionName = "zuh3-appservice-extension-k8se-build-service";
}
```

Reply:
this is for local debug. As when there is no env var found, a exception will be raised, but we could fallback to known value for debug.

Reply:
It will not throw exception even the variable is not existed, and will simply return Null. The exception comes from try to ToString() for Null.

Reply:
see. will update soon.